pkgs/development/python-modules/git-dummy/default.nix

```
@@ -0,0 +1,47 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  installShellFiles,
+  setuptools,
+  gitpython,
+  typer,
+  pydantic-settings,
+}:
+
+buildPythonPackage rec {
+  pname = "git-dummy";
+  version = "0.1.2";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "initialcommit-com";
+    repo = "git-dummy";
+    rev = "v${version}";
+    hash = "sha256-viybxn2J7SO7NgSvjwlP+tgtm+H6QrACafIy82d9XEk=";
+  };
+
+  build-system = [ setuptools ];
+
+  dependencies = [
+    gitpython
+    typer
+    pydantic-settings
+  ];
+
+  nativeBuildInputs = [ installShellFiles ];
+
+  postInstall = ''
+    installShellCompletion --cmd git-dummy \
+      --bash <($out/bin/git-dummy --show-completion bash) \
+      --fish <($out/bin/git-dummy --show-completion fish) \
+      --zsh <($out/bin/git-dummy --show-completion zsh)
+  '';
+
+  meta = with lib; {
+    homepage = "https://github.com/initialcommit-com/git-dummy";
+    description = "Generate dummy Git repositories populated with the desired number of commits, branches, and structure";
+    license = licenses.gpl2;
```

```suggestion
    license = licenses.gpl2Only;
```
`lincenses.gpl2` is deprecated.

Could you ask the upstream to confirm the licence?
This line may be wrong.
https://github.com/initialcommit-com/git-dummy/blob/dc85f631b06385c0f03d112ba3c0cb35162c2ab1/setup.py#L18

---

I have opened an issue to confirm, but I believe it is meant to be GPLv2 https://github.com/initialcommit-com/git-dummy/issues/5

---

Its gpl2 on master now https://github.com/initialcommit-com/git-sim/blob/main/pyproject.toml#L27